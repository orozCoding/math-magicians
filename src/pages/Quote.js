import React from 'react';

const Quote = () => (
  <div className="quoteContainer">
    <p className="quoteText">
      Mathematics is not about numbers, equations, computations, or algorithms:
      it is about understanding. - William Paul Thurston
    </p>
  </div>
);

export default Quote;
