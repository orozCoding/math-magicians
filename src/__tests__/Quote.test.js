import React from 'react';
import '@testing-library/react';
import renderer from 'react-test-renderer';
import Quote from '../pages/Quote';

describe('Renders Quote', () => {
  it('renders', () => {
    const tree = renderer
      .create(
        <Quote />,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
