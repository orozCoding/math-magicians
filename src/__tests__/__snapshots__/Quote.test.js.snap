// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Quote renders 1`] = `
<div
  className="quoteContainer"
>
  <p
    className="quoteText"
  >
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding.
  </p>
  <p
    className="quoteText"
  >
    - William Paul Thurston
  </p>
</div>
`;
